web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -29,27 +29,24 @@ import TextField from '@mui/material/TextField';
 import Tooltip from '@mui/material/Tooltip';
 import Typography from '@mui/material/Typography';
 
-import { styled } from '@mui/material/styles';
+import {styled} from '@mui/material/styles';
```

Comment:
You'll want to install prettier to take advantage of the default formatter we've configured, and reformat the JSX files. Other than that, looks good to me.